eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeParseMojo.java

```diff
@@ -112,6 +114,20 @@ public final class BinarizeParseMojo extends SafeMojo {
      */
     private Names names;
 
+    /**
+     * Map that matches ffi insert xpath to building of FFINode.
+     */
+    private final Map<String, Function<XML, FFINode>> factory = new MapOf<>(
```

Comment:
@levBagryansky can we do it static?

Reply:
@maxonfjvipon no since we use `this` in definition